packages/client/src/shared/libs/tests/socket-test.ts

```diff
@@ -0,0 +1,126 @@
+import { io } from 'socket.io-client';
+
+export default class Tester {
+  socketObject: any;
+
+  constructor(socketObject: any) {
+    this.socketObject = socketObject;
+  }
+
+  async connectTimeTest(type: 'websocket' | 'socket-io', serverUrl: string) {
+    if (type === 'websocket') {
```

Comment:
type이 `websocket`과 `socket.io` 밖에 없으니까 if-else 말고, if 문 내에서 return해서 사용하는 게 훨씬 깔끔할 거 같습니다!
```suggestion
    if (type === 'websocket') {
      ...
      return;
    }
    ... (socket.io 코드)
```